library/src/scala/runtime/quoted/Unpickler.scala

```diff
@@ -8,7 +8,7 @@ object Unpickler {
   /** Representation of pickled trees. For now it's String, but it
```

Comment:
Comment is wrong

Reply:
I'd probably make it a List[String] instead. SeqLiterals are expensive but `::` is cheap. So single string quotes could be represented as `quoted :: Nil`